src/plugins/git/loadRepository.test.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+
+import fs from "fs";
+import mkdirp from "mkdirp";
+import path from "path";
+import tmp from "tmp";
+
+import type {GitDriver} from "./loadRepository";
+import {localGit, loadRepository} from "./loadRepository";
+
+const cleanups: (() => void)[] = [];
+afterAll(() => {
+  cleanups.forEach((f) => {
+    f();
+  });
+});
+
+function mkdtemp() {
+  const result = tmp.dirSync();
+  cleanups.push(() => result.removeCallback());
```

Comment:
looks like this would be a really nice case to use python context managers (if we were in python) :)

Reply:
Yep: `with tempfile.TemporaryDirectory as tmpdir():` is great.